namespace cpp entity
namespace py ccentity.spirit
namespace as3 com.dreamfire.sanguo.data.definition.entity

////////////////////////////////////////////////
// 精灵外观类型定义
enum SpiritFacadeType {
  MIN = 0,

  HORSE = 0,                         // 野兽外观
  WING,                              // 翅膀外观

  MAX,
}


// 符文属性效果加成值类型
enum RuneAttributeType {
  MIN = 0, 

  PHYSIQUE = 0,                           // 体质
  STRENGTH,                               // 力量
  INTELLIGENCE,                           // 智力
  AGILE,                                  // 敏捷
  SPIRIT,                                 // 精神

  MAX,
}



